.gitignore

```diff
@@ -33,10 +33,9 @@ node_modules/
 !var/encore/.gitkeep
 /vendor/
 /web/assets/app/*
-!/web/assets/app/build/
 /web/assets/build/*
 /web/assets/ezplatform/*
-!/web/assets/ezplatform/build/
+/web/assets/translations/*
```

Comment:
Why not just ignore the entire `web/assets` and be done with it?

Reply:
Done in a5a99eeb8e2262cb623bbafcb9c748406ada0e3e